Cargo.lock

```diff
@@ -6960,7 +6972,7 @@ name = "twox-hash"
 version = "1.5.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
- "rand 0.7.2 (registry+https://github.com/rust-lang/crates.io-index)",
+ "rand 0.3.23 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
A bit unexpected version change in the dep.

Reply:
I would guess that this is because of https://github.com/rust-lang/cargo/issues/5702